client/src/components/AuditAdmin/Progress/JurisdictionDiscrepancies.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import { Classes, Colors, Dialog, H6, HTMLTable } from '@blueprintjs/core'
+import styled from 'styled-components'
+import {
+  IJurisdiction,
+  useDiscrepanciesByJurisdiction,
+} from '../../useJurisdictions'
+import useContestsJurisdictionAdmin from '../../JurisdictionAdmin/useContestsJurisdictionAdmin'
+import { IContest } from '../../../types'
+
+const ContestDiscrepanciesTable = styled(HTMLTable).attrs({
+  bordered: true,
+  striped: true,
+})`
+  &.${Classes.HTML_TABLE} {
```

Comment:
Nit: I don't think you need the `${Classes.HTML_TABLE}` in the selectors here, since styled-components will be creating its own class that applies only to this component.